src/provider.test.jsx

```diff
@@ -30,20 +28,17 @@ const clientList = {
 };
 
 describe('React Apollo Multiple Clients', () => {
-  /* eslint-disable no-console */
   beforeAll(() => {
     global.consoleError = console.error;
-    console.error = () => {};
+    console.error = () => null;
```

Comment:
Why is this changed to null?

Reply:
This is just to prevent the error to be shown while test suite is running. I could have done this explicitly for the specific tests instead of "all" ;)